README.md

```diff
@@ -38,11 +37,19 @@ export default LambdaWrapper({}, (di, request, done) => {
 
 ## Serverless Offline & SQS Emulation
 
-Serverless Offline only emulates API Gateway and Lambda, so publishing an SQS message would use the real SQS queue and trigger the consumer function (if any) in AWS. When working with offline code, the expectation is that the local functions will be invoked instead.
+Serverless Offline only emulates API Gateway and Lambda, so publishing an SQS message would use the real SQS queue and trigger the consumer function (if any) in AWS. When working with offline code, you often want the local functions to be invoked instead.
 
-In order to emulate SQS, `SQSService.prototype.publish` will check `DependencyInjection.prototype.isOffline`. If it's `true`, a Lambda client will be created and the message will be delivered to the offline Lambda endpoint, effectively running the consumer function _immediately_ as part of the original Lambda invocation. This works very well in the offline environment because invoking a Lambda function will trigger its whole (local) execution tree.
+Offline SQS behaviour can be configured by setting the `LAMBDA_WRAPPER_OFFLINE_SQS_MODE` environment variable. Available modes are:
 
-### How to use it?
+- `lambda` (the default): send messages to an offline Lambda endpoint
+- `sqs`: send messages to an offline SQS service, such as Localstack
+- `none`: no special handling of SQS offline; messages will be sent to AWS
```

Comment:
Since we really are sending to AWS SQS, I think it makes sense to call this `sqs`

```suggestion
- `sqs`: no special handling of SQS offline; messages will be sent to AWS
```

Reply:
Yeah, `sqs` or `aws` seem like good values here.